rest/endpoints/user_recordings/profile_recordings.yml

```diff
@@ -71,9 +77,48 @@ paths:
           example: 1526648593
           schema:
             type: integer
+        - in: query
+          name: filter[starts_after]
+          required: false
+          description: Include only recordings having broadcast start time after specified time (in Unix timestamp).
+          example: 1526648593
+          schema:
+            type: integer
+        - in: query
+          name: filter[ends_before]
+          required: false
+          description: Include only recordings having broadcast end time before specified time (in Unix timestamp).
+          example: 1526648593
+          schema:
+            type: integer
+        - in: query
+          name: filter[ends_after]
+          required: false
+          description: Include only recordings having broadcast end time after specified time (in Unix timestamp).
+          example: 1526648593
+          schema:
+            type: integer
+        - in: query
+          name: filter[series_id]
```

Comment:
Perhaps `filter[series.id]`? Series is a relationship of Recording, but the ID is an attribute of the Series resource.